src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.EasyRequest.cs

```diff
@@ -71,7 +71,10 @@ internal void InitializeCurl()
                 SetVerb();
                 SetDecompressionOptions();
                 SetProxyOptions(_requestMessage.RequestUri);
```

Comment:
In case of Windows, CredentialCache.DefaultCredentials are also used for the proxy if HttpClientHandle.Proxy == null